src/interfaces.ts

```diff
@@ -52,7 +52,7 @@ export interface DatadogLambdaProps {
  * For backward compatibility. It's recommended to use DatadogLambdaProps for
  * users who want to add Datadog monitoring for Lambda functions.
  */
-export type DatadogProps = DatadogLambdaProps;
+export interface DatadogProps extends DatadogLambdaProps {}
```

Comment:
There won't be a class `DatadogProps` in the generated Python package if we use `export type DatadogProps` here. The will be such a Python class if we use `export interface DatadogProps` here.